src/components/Content.tsx

```diff
@@ -1,15 +1,65 @@
 import React from 'react';
+import SplitPane from 'react-split-pane';
 
 import { Editor } from './Editor/Editor';
+import { Navigation } from './Navigation';
 import { Template } from './Template';
 
+import { debounce } from '../helpers';
+import state from '../state';
+
 interface ContentProps {}
 
 export const Content: React.FunctionComponent<ContentProps> = () => {
+  const sidebarState = state.useSidebarState();
+
+  const navigationEnabled = sidebarState.panels.navigation.get();
+  const editorEnabled = sidebarState.panels.editor.get();
+  const templateEnabled = sidebarState.panels.template.get();
+
+  const navigationAndEditor = (
+    <SplitPane
+      minSize={220}
+      maxSize={360}
+      pane1Style={!navigationEnabled ? { width: '0px' } : { overflow: 'auto' }}
+      pane2Style={!editorEnabled ? { width: '0px' } : undefined}
+      primary={!editorEnabled ? 'second' : 'first'}
```

Comment:
@magicmatatjahu generally I don't mark all cases in PR reviews, but require you to search for them 😄

Reply:
I also changed them :)